NEW_MODULE_TECH_EVAL.MD

```diff
@@ -80,12 +93,16 @@ Information gathered includes:
 1. If applicable, the module developers resolve all issues and request a re-review.  
     * * JIRA workflow: the ticket is transitioned from  **REJECTED** to **SUBMITTED**
 
-### Roles
+## Process Feedback
+
+1. TC asks the submitter if they have any feedback on the New Module Technical Evaluation process.
+    * TC considers that feedback as part of the next process evaluation improvement round.
+
```

Comment:
@mweaver-ebsco noted that the `TCR Process Improvements` section on the evaluation template is too early for the submitter to give feedback on the evaluation & decision processes.  So this explicitly provides that opportunity.